orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/tasks/CreateBakeTask.groovy

```diff
@@ -140,7 +140,7 @@ class CreateBakeTask implements RetryableTask {
     }
 
     if (!roscoApisEnabled) {
-      bakeRequest = bakeRequest.copyWith(templateFileName: null, extendedAttributes: null)
+      bakeRequest = bakeRequest.copyWith(templateFileName: null, extendedAttributes: null, allowMissingPackageInstallation: null)
```

Comment:
`allowMissingPackageInstallation` should be elided whether `roscoApisEnabled` is enabled or not. We don't want to propagate the new bake request attribute `allowMissingPackageInstallation` from orca to the bakery whether it is Rosco or the internal Netflix bakery.